.github/workflows/ci.yml

```diff
@@ -12,17 +12,18 @@ jobs:
       matrix:
         javaversion: ["8", "9", "10", "11", "12", "13", "14", "15", "16", "17"]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up Java ${{ matrix.javaversion }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
```

Comment:
Let's bump this now to v3 and see if builds pass.

Reply:
build passed, it works with `zulu` for now